openlibrary/plugins/upstream/utils.py

```diff
@@ -1222,13 +1222,14 @@ def _get_author_config():
 
     The results are cached on the first invocation.
     Any changes to /config/author page require restarting the app.
-
     """
-    thing = web.ctx.site.get('/config/author')
-    if hasattr(thing, "identifiers"):
-        identifiers = [Storage(t.dict()) for t in thing.identifiers if 'name' in t]
-    else:
-        identifiers = {}
+    # Load the author config from the author.yml file in the author directory
+    with open('openlibrary/plugins/openlibrary/config/author/author.yml') as in_file:
```

Comment:
```suggestion
    with open('openlibrary/plugins/openlibrary/config/author/identifiers.yml') as in_file:
```
Update name of author identifier configuration file to follow same pattern as the edition identifier config file.